src/jdk.httpserver/share/classes/com/sun/net/httpserver/Authenticator.java

```diff
@@ -92,15 +90,15 @@ public int getResponseCode() {
         /**
          * Creates a {@code Success} instance with given {@code Principal}.
          *
-         * @param p The authenticated user you wish to set as Principal
+         * @param p the authenticated user you wish to set as {@code Principal}
          */
         public Success (HttpPrincipal p) {
             principal = p;
         }
         /**
          * returns the authenticated user Principal
```

Comment:
`* Returns the authenticated user {@code Principal}.`

Reply:
Change made in https://github.com/openjdk/jdk/pull/301/commits/604ea45ca36c8fb0cb195c581f822dc5ae1c23c1